api/integrations/segment/serializers.py

```diff
@@ -1,10 +1,23 @@
+from rest_framework import serializers
+
 from integrations.common.serializers import (
     BaseEnvironmentIntegrationModelSerializer,
 )
+from integrations.segment import constants
 from integrations.segment.models import SegmentConfiguration
 
 
 class SegmentConfigurationSerializer(BaseEnvironmentIntegrationModelSerializer):
+
+    base_url = serializers.ChoiceField(
+        choices=[
+            constants.DEFAULT_BASE_URL,
+            constants.DUBLIN_BASE_URL,
+        ],
```

Comment:
I don't enjoy the thought of having to generate a migration each time this list changes on Segment side.

Reply:
Migration? This is just in the serializers code.

Reply:
Apologies — I stand corrected.